mgr/manager.go

```diff
@@ -312,9 +312,17 @@ func (m *Manager) Load(forced bool) error {
 		manager := dummyMgr
 		if cert.Action != "" && cert.Action != "nop" {
 			manager, err = svcmgr.New(s, cert.Action, cert.Service)
+			if err != nil {
+				return err
+			}
 		}
-		if err != nil {
-			return err
+		// If action is undefined and svcmgr isn't dummy, we will throw a warning due to likely undefined cert renewal behavior
```

Comment:
What's your thoughts on moving the validation into svcmgr.New?  Specifically having it ask the service manager implementation to self validate?

Doing it that way would allow for this logic to mostly be buried, and strict would just become an error check against the error returned.

Thoughts?

Reply:
I don't know if that makes sense because from what I can tell the certificate specs haven't been loaded yet in svcmgr.New(). So how would I be able to tell what the certificate action is? 

Plus, if I do the check where the cert specs are actually loaded then I can prevent one from being added to the tracked certs array. If I do the check in svcmgr.New(), then I'd have to iterate through all the certificates, check for no-action+not-dummy, and modify the certs array there, which isn't particularly clean either (but also totally doable)

Reply:
Fair; code has to be restructured to support what I'm asking for, so I'll follow up after I do some more refactoring.

Reply:
I'll follow up on this; there is a way to do it but requires some restructuring.

Reply:
Fair.  I'll see if the next bit of refactoring can add the structure I'm suggesting, but until then let's merge this.

Reply:
Nice to see github doesn't synchronize message threads between the commit and overview pane.  Pardon the double messages, I legitimately am blaming github here.

Not a good sign that I'm missing bitbucket after the last 2 days of doing work in github....